src/components/molecules/TableHeader/components/EditTableTitleModal/EditTableTitleModal.tsx

```diff
@@ -41,6 +42,7 @@ export const EditTableTitleModal: React.FC<EditTableTitleModalProps> = ({
   onHide,
 }) => {
   const venueId = useVenueId();
+  const [isLoading, setIsLoading] = useState<boolean>(false);
```

Comment:
[NB] Can this be rewritten using `showHide` hook? Looks like it's a good case for using it

Reply:
It can. However, is it okey to do so. When the name implies that it's show/hide but in my case is loading or not

Reply:
I suppose, I can create a hook that's only used for loading.

Reply:
It's handled like this

![image](https://user-images.githubusercontent.com/63194656/120025498-6c51e380-bff9-11eb-9aa9-7ec286dc46e2.png)

Reply:
But the benefit is almost non existent. 
Instead of:
```tsx
const [isLoading, setIsLoading] = useState<boolean>(false);
```
It will be:
```tsx
const { isLoading, setIsLoading } = useLoading();
```

It's still going to be one line code.

Reply:
Actually useAsyncFn returns `loading` I will use that instead.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1473/commits/5ca74272522dfcd2eae36fa854d48158194d67e0

Reply:
I think the `useAsyncFn`'s out of the box loading is better approach. If you disagree I will use `useShowHide`

Reply:
Agreed RE: useAsyncFn

Reply:
With useShowHide, we need a better name for it that allows it to be used more generically tbh.